app/static/css/dark_mode_override.css

```diff
@@ -0,0 +1,34 @@
+/*
+    Django admin styles
+    extended from https://github.com/django/django/blob/main/django/contrib/admin/static/admin/css/dark_mode.css
+    Remember to sync colours with front-end CSS.
+*/
+
+@media (prefers-color-scheme: dark) {
+    :root {
+      --primary: #2d4481;
+      --primary-black: #000000;
+      --primary-white: #fff;
+
+      --link-fg: var(--primary-white);
```

Comment:
A white link colour unfortunately makes some links look exactly like other text, so I wanted to hear why you don't want it to be the primary colour like in the light mode? Look for example at the link in this screenshot:
![prent](https://github.com/SADiLaR/term_platform/assets/1551995/8366fbca-abe6-4f89-bde6-afb4f2083eb1)
The one help text has a link, the other one is plain text. You can't distinguish between them visually, and I think the white link colour is the cause.

Reply:
This is address with the last push